docs/features/intervals.md

```diff
@@ -0,0 +1,34 @@
+<script setup>
+
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+</script>
+
+# Intervals <VersionBadge pr="2075" />
+
+These helper functions are provided for convenience as a **tick** option for [scales](./scales.md), as the **thresholds** option for a [bin transform](../transforms/bin.md), or other use. See also [d3-time](https://d3js.org/d3-time).
+
+## numberInterval(*period*) {#numberInterval}
+
+```js
+Plot.numberInterval(2)
+```
+
+Given a number *period*, returns a corresponding range interval implementation. The returned interval implements the *interval*.range, *interval*.floor, and *interval*.offset methods.
```

Comment:
We should document what these functions do, in particular for a negative _period_. I also wonder if we'd want to implement *interval*.ceil for parity with date intervals?

Reply:
On the latter, I don’t see that we need to do that (especially as part of this PR) since we already differentiate between `RangeInterval` and `NiceInterval`. Isn’t it good enough as is? We’ve got a lot of other stuff to do.

I can document the range methods in the Markdown. They are already documented in TypeScript.